### PR TITLE
Sales Assistant Blog 

### PR DESCRIPTION
**Pull request by KeinTung**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: jekyll/_posts/2021-11-11-sales-assistant-bot.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+---
+Kevin Tung sales assistant initial report
+  
+![Wechaty Logo][wechaty-logo-image]
+
+<center class="half"><center/>
+  <img src="/Users/kevintung/workspace/juzi/asset/01.webp" style="zoom: 21%;" />
+  <img src="/Users/kevintung/workspace/juzi/asset/02.webp" alt="sales-Wechat" style="zoom:21%;" />
+</center>
+
```
Comment (lijiarui) [inline jekyll/_posts/2021-11-11-sales-assistant-bot.md]: Please use markdown to insert images into the blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please:

1. resolve the conflict and make CI green
2. add video intro in this blog

File: jekyll/_posts/2021-11-11-sales-assistant-bot.md (hunk 1 above)
Comment (KeinTung) [reply jekyll/_posts/2021-11-11-sales-assistant-bot.md]: Fixed

**Comment by KeinTung** [comment]
> Please:
> 
> 1. resolve the conflict and make CI green
> 2. add video intro in this blog

1. I resolved the conflict
2. Can I post this initial blog first?  I will then add my video intro to my mid-term blog.

File: package.json [hunk 2]
```diff
@@ -74,5 +74,8 @@
     "scripts": {
       "pre-push": "npx git-scripts-pre-push"
     }
+  },
+  "dependencies": {
+    "file-name-linter": "^1.1.0"
```
Comment (huan) [inline package.json]: I don't think this modification is necessary?

Comment (KeinTung) [reply package.json]: yes, I should delete this

**Review by KeinTung — COMMENTED** [review]
resolved conflicts

File: jekyll/_contributors/kevintung.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: kevintung
+site: https://github.com/KevinTung
+avatar: /assets/contributors/kevintung/avatar.webp
+bio: hi~
```
Comment (qhduan) [inline jekyll/_contributors/kevintung.md]: 这里还是改认真一点

File: jekyll/_posts/2021-11-11-sales-assistant.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: sales-assistant 项目规划
+author: kevintung
+categories: project
+tags: productivity
+image: /assets/2021/11-sales-assistant/sales-meme.webp
+---
+
+销售在跟客户沟通的过程中，会出现回复客户不及时，以及回复时存在错别字和病句的现象。所以HR部门要利用机器人记录这种情况，用于计算销售人员的绩效。
+<!-- ![01](/assets/2021/11-sales-assistant/01.webp)
```
Comment (qhduan) [inline jekyll/_posts/2021-11-11-sales-assistant.md]: 已经不需要的注释删一下

**Review by qhduan — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/kevintung.md (hunk 3 above)
Comment (KeinTung) [reply jekyll/_contributors/kevintung.md]: fixed

File: jekyll/_posts/2021-11-11-sales-assistant.md (hunk 4 above)
Comment (KeinTung) [reply jekyll/_posts/2021-11-11-sales-assistant.md]: fixed

**Comment by KeinTung** [comment]
I found that I mistakenly merged the main repo into my repo.  After a hard reset, I found that I could only create a new branch , sales-mid-term, and created a new PR.

**Pull request closed without merging**